File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/04_compito_java_2015_06_23/Scritto.java
public class Scritto extends Appello {
    private String aula;
    private int durata;

    public Scritto(int codice, String docente, String corso, int giorno, int mese, int anno, int ora, String aula,
            int durata) {
        super(codice, docente, corso, giorno, mese, anno, ora);
        this.aula = aula;
        this.durata = durata;
    }

    public String toString() {
        return "scritto" + "\t" + codice + "\t" + docente + "\t" + corso + "\t" + giorno + "\t" + mese + "\t" + anno
                + "\t" + ora + "\t" + aula + "\t" + durata + "\t-";
    }
}

File: II - Linguaggi di programmazione e laboratorio/Esame/Python/04_compito_java_20150623/Scritto.py
from Appello import Appello


class Scritto(Appello):
    def __init__(self, tipo, codice, docente, corso, giorno, mese, anno, ora, aula, durata):
        super().__init__(tipo, codice, docente, corso, giorno, mese, anno, ora)
        self._aula = aula
        self._durata = durata

    def __str__(self):
        return "Scritto\t" + super().__str__() + "\t" + self._aula + "\t" + str(self._durata) + "\t-"
